l10n_be_iso20022_pain/models/account_move.py

```diff
@@ -25,7 +25,8 @@ class AccountMove(models.Model):
     _inherit = "account.move"
 
     reference_type = fields.Selection(
-        selection_add=[("bba", "BBA Structured Communication")]
+        selection_add=[("bba", "BBA Structured Communication")],
```

Comment:
@sbejaoui Is this still needed following this:

https://github.com/odoo/odoo/blob/ec5f6003f8f6f8de16c6c8b2b023a0511a55d498/addons/account/models/account_move.py#L2605

And this:

https://github.com/odoo/odoo/blob/ec5f6003f8f6f8de16c6c8b2b023a0511a55d498/addons/l10n_be/models/account_move.py#L20C9-L20C42

Reply:
Yes, in the previous versions, `_get_invoice_reference_be_partner` was located in `l10n_be_invoice_bba`, but now it has been merged into `l10n_be`. This method provides a default BBA-formatted reference.

`l10n_be_iso20022_pain` add a new reference type `bba` to ensure that the reference (whether generated or edited by the user) complies with the valid BBA format.

Reply:
Correct, 

I believe we should keep only the check for the ref in this module.

cc @ThomasBinsfeld